libs/ballot-interpreter-nh/src/layout.ts

```diff
@@ -309,10 +309,20 @@ export function generateBallotPageLayouts(
       });
     }
 
+    // ensure the layouts are returned in the contest-order defined by the election definition
+    const contestsInOrder: BallotPageContestLayout[] = election.contests
+      .map((electionContest) =>
+        contests.find((c) => c.contestId === electionContest.id)
+      )
+      .filter(
+        (contestLayout): contestLayout is BallotPageContestLayout =>
+          contestLayout !== undefined
+      );
```

Comment:
Another solution ⛳ 
```ts
    const contestsInOrder = [...contests].sort(
      (a, b) =>
        election.contests.findIndex(({ id }) => id === a.contestId) -
        election.contests.findIndex(({ id }) => id === b.contestId)
    );
```

Reply:
ahah nice, that is more elegant :) for a future patch!